src/Components/Publishing/ToolTip/LinkWithTooltip.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react"
+import { ToolTip } from "./ToolTip"
+import { OverlayTrigger } from "../../OverlayTrigger"
+import PropTypes from "prop-types"
+import { Gene } from "../Fixtures/Components"
+
+interface Props {
+  url: string
+  node: any
+}
+
+interface State {
+  show: boolean
+}
+
+export class LinkWithTooltip extends Component<Props, State> {
+  static contextTypes = {
+    tooltipsData: PropTypes.object,
+  }
+
+  state = {
+    show: false,
+  }
+
+  render() {
+    const { show } = this.state
+    const toolTip = (
+      <div>
+        <ToolTip entity={Gene} model="gene" />
+      </div>
+    )
+
+    if (this.context.tooltipsData) console.log(this.context.tooltipsData)
```

Comment:
This gives us access to all data fetched from metaphysics using relay so we can look up the data object based on the url